src/Widgets/LoginPage.vala

```diff
@@ -84,6 +72,35 @@ public class FeedReader.LoginPage : Gtk.Stack {
 		reset();
 	}
 
+	private void RefreshPlugins()
+	{
+		var children = m_accountList.get_children();
+		foreach(Gtk.Widget row in children)
+		{
+			m_accountList.remove(row);
+			row.destroy();
+		}
+
+		FeedServer.get_default().getPlugins().foreach((extSet, info, ext) => {
+			var plug = ext as FeedServerInterface;
+			if(plug != null)
+			{
+				Logger.debug("LoginPage: add plugin " + plug.getID());
+				BackendInfo pluginfo = BackendInfo()
+				{
+					ID = plug.getID(),
+					name = plug.serviceName(),
+					flags = plug.getFlags(),
+					website = plug.getWebsite(),
+					iconName = plug.iconName()
+				};
+				m_accountList.add(new LoginRow(pluginfo));
+			}
```

Comment:
Should we log if the cast to `FeedServerInterface` fails?

Reply:
At least it wouldn't hurt.